src/components/common/Footer.tsx

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import "./Footer.css";
+import fw from "@assets/svg/FW200.svg";
+import instagram from "../../assets/images/instagram_logo.png";
+
+const Footer = () => {
+	return (
+		<div className="footer">
+			<div className="footer section__padding">
+				<div className="footer-links">
+					<div className="footer-links-div">
+						<div className="logo">
+							<img src={fw} alt="fw" className="logo-image" />
+						</div>
+					</div>
+					<div className="footer-links-div">
+						<a href="/contribute">
+							<p>Contribute</p>
```

Comment:
instead of `<p>` elements we should use the  MUI given Typography component. We could discuss the details during our next meeting. An example, in this case, would be:
```ts
<Typography
  variant="body1"
>
    Contribute
</Typography>
```